apiserver/facades/client/charms/client_integration_test.go

```diff
@@ -56,27 +55,3 @@ func (s *clientMacaroonIntegrationSuite) TestAddLocalCharmWithFailedDischarge(c
 	c.Assert(err, gc.ErrorMatches, `Put https://.+: cannot get discharge from "https://.*": third party refused discharge: cannot discharge: login denied by discharger`)
 	c.Assert(savedURL, gc.IsNil)
 }
-
-func (s *clientMacaroonIntegrationSuite) TestAddLocalCharmSuccess(c *gc.C) {
```

Comment:
If we remove this we need a bash test. Can you make a task for this.

Reply:
I definitely will, I'm just finishing fixing an issue at resolve charm download and will do it right after.

Reply:
Here it is: https://warthogs.atlassian.net/browse/JUJU-7362